src/main/webapp/WEB-INF/static/js/trialmanager/sample-genotype/sample-genotype-modal.js

```diff
@@ -44,18 +44,22 @@
 
 			var genotypeModalService = {};
 
-			genotypeModalService.openImportGenotypesModal = function (listId) {
+			genotypeModalService.openImportGenotypes = function (listId, fromFile) {
```

Comment:
Instead of using parameter fromFile, just create another method to show Import Genotypes from File screen/modal